CMakeLists.txt

```diff
@@ -30,14 +30,27 @@
 if(NOT ${ENABLE_SQLITE})
   message(STATUS "ORAN requires sqlite3 library")
   return()
+else()
+  set(sqlite_libraries
+      ${SQLite3_LIBRARIES}
+  )
 endif()
 
-find_external_library(DEPENDENCY_NAME Torch
-                      HEADER_NAME
-                        torch/script.h
-                        torch.h
-                      LIBRARY_NAME torch
-                      SEARCH_PATHS $ENV{LIBTORCHPATH})
+if(DEFINED ENV{LIBTORCHPATH})
+  find_external_library(DEPENDENCY_NAME Torch
+                        HEADER_NAME
+                          torch/script.h
+                          torch.h
+                        LIBRARY_NAME torch
+                        SEARCH_PATHS $ENV{LIBTORCHPATH})
+else()
+  find_external_library(DEPENDENCY_NAME Torch
+                        HEADER_NAME
+                          torch/script.h
+                          torch.h
+                        LIBRARY_NAME torch
+                        SEARCH_PATHS "")
```

Comment:
If we're not providing any paths to search, the `SEARCH_PATHS` argument is optional, so just remove it

Reply:
Resolved by 58dc507c4b4bd8af72adc667c55113329b4f536a